tests/scripts/all.sh

```diff
@@ -403,7 +423,19 @@ scripts/config.pl unset MBEDTLS_THREADING_C
 scripts/config.pl unset MBEDTLS_MEMORY_BACKTRACE # execinfo.h
 scripts/config.pl unset MBEDTLS_MEMORY_BUFFER_ALLOC_C # calls exit
 scripts/config.pl unset MBEDTLS_PLATFORM_TIME_ALT # depends on MBEDTLS_HAVE_TIME
-CC=armcc AR=armar WARNING_CFLAGS= make lib
+
+CC="$ARMC5_CC" AR="$ARMC5_AR" WARNING_CFLAGS= make lib
```

Comment:
I agree with this comment. However, I would also like to point out that there is a separate PR https://github.com/ARMmbed/mbedtls/pull/585 that already does this to some extent. I consider that we should do this change as a separate PR or add to the existing PR https://github.com/ARMmbed/mbedtls/pull/585 rather than introduce the changes here.